coordinator/src/orderbook/routes.rs

```diff
@@ -216,6 +216,10 @@ async fn websocket(stream: WebSocket, state: Arc<AppState>) {
                                     tracing::error!("Could not respond to user {e:#}");
                                     return;
                                 }
+
+                                let mut authenticated_users =
+                                    state.authenticated_users.lock().await;
+                                authenticated_users.insert(pubkey, local_sender.clone());
```

Comment:
🤔 We insert them here - should we also remove them when they e.g. disconnect?

Reply:
Let's handle this when we want to send to the client, because that's when we will figure out if the client was disconnected. 

This however means that this hashmap will grow with connected users: If we never match them we will never notice that they may have dissapeared and we will not remove them from the list. 

I think this is a problem to deal with in the future :)